README.md

```diff
@@ -1,34 +1,63 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# SILACAK
```

Comment:
```suggestion
# Silacak
```

Nitpick: Silacak (sentence case) instead of all uppercase, as dictated by the new design.